site/pages/api/publish-edit.test.ts

```diff
@@ -302,4 +319,42 @@ describe("publishEdit", () => {
             expect(insertDisruptionSpy.mock.calls[0]).toMatchSnapshot();
         },
     );
+
+    it("should send an email to org admin if an org staff creates a disruption", async () => {
+        getDisruptionSpy.mockResolvedValue(disruptionWithConsequences);
+        getSessionSpy.mockImplementation(() => ({ ...mockSession, isOrgStaff: true, isSystemAdmin: false }));
+        getAllUsersInGroupSpy.mockResolvedValue(mockOrgAdmins);
+        const { req, res } = getMockRequestAndResponse({
+            body: {
+                disruptionId: defaultDisruptionId,
+            },
+            mockWriteHeadFn: writeHeadMock,
+        });
+
+        await publishEdit(req, res);
+
+        expect(sesMock.send.calledOnce).toBeTruthy();
+        expect(sesMock.commandCalls(SendEmailCommand)[0].args[0].input.Destination).toEqual({
+            ToAddresses: ["emailtoshow@test.com"],
+        });
+    });
+
+    it("should redirect to dashboard and log an error if an org staff creates a disruption an no org admins exists for staff member's org", async () => {
```

Comment:
If an org admin is not yet setup - should the disruption be impacted? The disruption can be in pending to approval status which can be approved by a publisher or approved later when an admin user is created?

Reply:
The disruption isn't impacted. If an org admin doesn't exist the disruption will be published still but an email won't be sent. We only log an error message to help with troubleshooting if needed.